src/sidebar/components/UserMenu.js

```diff
@@ -92,9 +91,9 @@ function UserMenu({ auth, frameSync, onLogout, settings }) {
     </span>
   );
   return (
-    // FIXME: KeyDown handling is temporary for Notebook "easter egg"
+    // Allow keyboard shortcut 'n' to open Notebook
```

Comment:
This was intended to be temporary but I find it's a handy keyboard shortcut to keep around.